dev/run_container_podman.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+set -euo pipefail
+
+PORTAINER_DATA=${PORTAINER_DATA:-/tmp/portainer}
+PORTAINER_PROJECT=${PORTAINER_PROJECT:-$(pwd)}
+PORTAINER_FLAGS=${PORTAINER_FLAGS:-}
+
+podman rm -f portainer
```

Comment:
should be a `sudo` command or it will try to remove the previous container from the user context, while the container is deployed on the root context (`/run/user/1000/podman/podman.sock` vs `/run/podman/podman.sock`)

Reply:
Good point, updated